modules/era/sql/era_mob_droplist.sql

```diff
@@ -15033,14 +15033,14 @@ INSERT INTO `mob_droplist` VALUES (1855,4,0,1000,1784,0);         -- Phuabo Orga
 INSERT INTO `mob_droplist` VALUES (1856,0,0,1000,1785,@COMMON); -- Xzomit Organ (Common, 15%)
 INSERT INTO `mob_droplist` VALUES (1856,0,0,1000,1855,@COMMON); -- High-Quality Xzomit Organ (Common, 15%)
 INSERT INTO `mob_droplist` VALUES (1856,0,0,1000,1783,@RARE);   -- Sample Of Luminian Tissue (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1856,0,0,1000,4104,@RARE);   -- Fire Cluster (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1856,0,0,1000,4105,@RARE);   -- Ice Cluster (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1856,0,0,1000,4106,@RARE);   -- Wind Cluster (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1856,0,0,1000,4107,@RARE);   -- Earth Cluster (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1856,0,0,1000,4108,@RARE);   -- Lightning Cluster (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1856,0,0,1000,4109,@RARE);   -- Water Cluster (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1856,0,0,1000,4110,@RARE);   -- Light Cluster (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1856,0,0,1000,4111,@RARE);   -- Dark Cluster (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (1856,1,1,@VRARE,4104,125);   -- Fire Cluster (Group 1 - Very Rare, 1% * 12.5%)
```

Comment:
https://ffxiclopedia.fandom.com/wiki/Om%27xzomit
Feels like it should be very rare.